src/raftstore/store/config.rs

```diff
@@ -358,6 +358,18 @@ impl Config {
         }
         Ok(())
     }
+
+    pub fn write_into_metrics(&self) {
+        let metrics = register_gauge_vec!(
+            "tikv_config_raftstore",
+            "Config information of raftstore",
+            &["name"]
+        )
+        .unwrap();
+        metrics
+            .with_label_values(&["consistency_check_interval_seconds"])
```

Comment:
How about other configurations of `raftstore`?

Reply:
We only output important configurations in this PR.

Reply:
Yes. But some metrics like `raft_base_tick_interval` may be also important?